sentry/src/test/java/io/sentry/instrumentation/file/SentryFileInputStreamTest.kt

```diff
@@ -274,6 +276,15 @@ class SentryFileInputStreamTest {
         assertEquals(false, fileIOSpan.data[SpanDataConvention.BLOCKED_MAIN_THREAD_KEY])
         assertNull(fileIOSpan.data[SpanDataConvention.CALL_STACK_KEY])
     }
+
+    @Test
+    fun `when tracing is disabled does not instrument the stream`() {
```

Comment:
should we add the same test in `SentryFileOutputStreamTest`?

Reply:
yeah I can add one!